source/chapters/appendix/mixxx_controls.rst

```diff
@@ -3768,15 +3769,25 @@ Controls
    :range: 0.0..1.0
 
 
-.. mixxx:control:: [EffectRack1_EffectUnitN],next_chain
-                   [EqualizerRack1_[ChannelI]],next_chain
-                   [QuickEffectRack1_[ChannelI]],next_chain
+.. mixxx:control:: [EffectRack1_EffectUnitN],next_chain_preset
+                   [EqualizerRack1_[ChannelI]],next_chain_preset
+                   [QuickEffectRack1_[ChannelI]],next_chain_preset
 
    Cycle to the next EffectChain preset after the currently loaded preset.
 
    :range: binary
 
 
+.. mixxx:control:: [EffectRack1_EffectUnitN],num_chain_presets
+                   [EqualizerRack1_[ChannelI]],num_chain_presets
+                   [QuickEffectRack1_[ChannelI]],num_chain_presets
+
+   The number of effect chain presets available in this EffectUnit, including the
+   empty/passthrough preset `---.
```

Comment:
typo?

Reply:
yup, thanks.
though, neither ` nor " allow keeping three separate dashes, it's always joint to em dash :shrug: